Dockerfile

```diff
@@ -1,62 +1,5 @@
-FROM tiangolo/uwsgi-nginx-flask:python3.7
-
-
+FROM gcr.io/neuromancer-seung-import/pychunkedgraph:graph-tool_dracopy
```

Comment:
Could you please add the Dockerfile for `graph-tool_dracopy` to the repo?

Reply:
that's here at the moment https://github.com/seung-lab/PyChunkedGraph/tree/base-docker-image
I'm not sure what's the best way to manage it, if I add it to master it will have to be maintained in `master` and `base-docker-image` branches
Any ideas?

Reply:
more specifically https://github.com/seung-lab/PyChunkedGraph/tree/base-docker-image/docker-base
created a separate branch because of travis cron job, though it would be clean to keep the cron and push/pr triggers separate

Reply:
and also because there can only be one `.travis.yml` which is the actual limiting factor